python/hail/keytable.py

```diff
@@ -636,6 +636,76 @@ def select(self, selected_columns):
 
     @handle_py4j
     @record_method
+    @typecheck_method(selected_fields=oneof(strlike, listof(strlike)),
```

Comment:
You should delete export_samples, export_variants and export_genotypes and use this + KeyTable.export.